charts/fission-all/templates/pre-upgrade-checks/pre-upgrade-job.yaml

```diff
@@ -41,5 +41,5 @@ spec:
         {{- if .Values.terminationMessagePolicy }}
         terminationMessagePolicy: {{ .Values.terminationMessagePolicy }}
         {{- end }}
-      serviceAccountName: fission-svc
```

Comment:
We should set hook-weight here so that service account is create first then deployment.
```
  helm.sh/hook-weight: "-5"
```

https://helm.sh/docs/topics/charts_hooks/#writing-a-hook